weatherman/All_tasks.py

```diff
@@ -0,0 +1,292 @@
+import os
+import sys
+
+class readings():
+    highest_temp = 0

```

Comment:
let's not make static variables and set these field in `__init__` so that new objects of this class doesn't discard previous values.